python/ray/workflow/api.py

```diff
@@ -264,8 +264,8 @@ def resume_async(workflow_id: str) -> ray.ObjectRef:
     Examples:
         >>> from ray import workflow
         >>> start_trip = ... # doctest: +SKIP
-        >>> trip = start_trip.step() # doctest: +SKIP
-        >>> res1 = trip.run_async(workflow_id="trip1") # doctest: +SKIP
+        >>> trip = start_trip.bind() # doctest: +SKIP
+        >>> res1 = workflow.run_async(trip, workflow_id="trip1") # doctest: +SKIP
         >>> res2 = workflow.resume("trip1") # doctest: +SKIP
```

Comment:
now it is `resume_async`, maybe also in other places

Reply:
I see. Thanks for catching!